jekyll/_cci2/jobs-steps.md

```diff
@@ -1,17 +1,21 @@
 ---
 layout: classic-docs
 title: "Jobs, Steps, and Workflows"
-short-title: "Jobs, Steps, and Workflows"
+short-title: "Orbs, Jobs, Steps, and Workflows"
 description: "Description of Jobs and Steps"
 categories: [migration]
 order: 2
 ---
 
-The document provides an overview of Jobs and Steps, new [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) keys that are nested inside Steps, and new keys for Workflows.
+The document provides an overview of Jobs, Steps, Workflows and new [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) keys for Orbs.
 
 * TOC
 {:toc}
 
+## Orbs Overview
+
+Orbs are packages of config (similar to Ruby Gems) that you either import by name or configure inline to simplify your config, share, and reuse config within and across projects. See [Using Orbs]({{ site.baseurl }}/2.0/using-orbs/) for details about how to use orbs in your config and an introduction to Orb design.
```

Comment:
I think we should be careful about comparing orbs so directly to gems, as gems are arbitrary code packages that have lots of implications in the world of ruby and lots of negative associations outside the ruby community. "packages of config" is probably sufficient without the comparison.

Reply:
Also, I think "Orb design" should probably be lower case "orb"

Reply:
I also liked how orbs were previously described as "_reusable_ packages of config," though I can see that for many folks "packages" already implies "reusable"

Reply:
thanks, removed part aobut ruby gems and lowercased orbs